_overviews/scala3-book/methods-most.md

```diff
@@ -256,13 +256,14 @@ A `match` expression can also be used as the entire method body, and often is.
 Here’s another version of `isTruthy`, written with a `match` expression :
 
 ```scala
-def isTruthy(a: Any) = a match
+def isTruthy(a: Matchable) = a match
   case 0 | "" => false
   case _ => true
 ```
 
-This method works just like the previous method that used an `if`/`else` expression.
+This method works just like the previous method that used an `if`/`else` expression. We use `Matchable` instead of `Any` as the parameter's type because it's a safer alternative that will still admit any concrete value or reference.
```

Comment:
What about the following?

```suggestion
This method works just like the previous method that used an `if`/`else` expression. We use `Matchable` instead of `Any` as the parameter's type to accept any value that supports pattern matching.
```